### ISSUE TITLE
about  data  inputs  when infer? trimap  in real time?

### ISSUE THREAD

**Issue by zhanghongyong123456** [Hongje/OTVM#1]
Thank you for Posting this project and the effect of the project presentation is truly amazing,I see that this project needs to enter trimap;In a real world scenario, it is difficult to capture a trimap in real time;How can this problem be solved.for example:One moment a dog, the next a table or something,How to capture the corresponding trimap in real time

**Comment by Hongje** [comment]
Hi, thanks for your interest in our work!
To run OTVM in a real-world scenario, a manually annotated trimap should be given for one frame. Examples of the annotated trimap in a real-world scenario can be found in our [Demo Video](https://youtu.be/qkda4fHSyQE?t=3) (00:03~01:34, in the video, the right upper trimap is the manually annotated trimap).
Since annotating trimap is manually processed in our system and requires some time, it is hard to capture the corresponding trimap in real-time.

**Comment by zhanghongyong123456** [comment]
> Hi, thanks for your interest in our work! To run OTVM in a real-world scenario, a manually annotated trimap should be given for one frame. Examples of the annotated trimap in a real-world scenario can be found in our [Demo Video](https://youtu.be/qkda4fHSyQE?t=3) (00:03~01:34, in the video, the right upper trimap is the manually annotated trimap). Since annotating trimap is manually processed in our system and requires some time, it is hard to capture the corresponding trimap in real-time.
If I need to test my own video, I need to use PS to do trimap first, and I cannot increase or decrease objects at any time for consecutive frames.If the foreground changes, i need to enter a new trimap to update

**Comment by Hongje** [comment]
That's right. You need to use a tool for annotating a trimap for one frame, such as PS. And if you want to increase or decrease objects at any time, or change foreground objects, then you need to annotate a new trimap.

> If I need to test my own video, I need to use PS to do trimap first, and I cannot increase or decrease objects at any time for consecutive frames.If the foreground changes, i need to enter a new trimap to update
